src/code-templates/libraries/logger/logger.ts

```diff
@@ -1,10 +1,16 @@
-import { configurationProvider } from "configuration-provider";
+import {configurationProvider} from "../configuration-provider";
+import {pino} from "pino";
 
-// example on how to read the level: configurationProvider.get("logger.level")
-export const info = (message) => {
-  console.log(message);
-};
 
-export const error = (message) => {
-  console.error(message);
-};
+const logFile = configurationProvider.get("logger.destination");
+export default pino(
```

Comment:
Maybe we should not export pino rather our own object to prevent clients from coupling directly into pino